e2e_tests/test_search.py

```diff
@@ -149,3 +176,44 @@ def test_click_on_clear_all_as_standard_user_redirects_to_browse_transferring_bo
         standard_user_page.get_by_role("link", name="Clear all").click()
 
         expect(standard_user_page).to_have_url(url)
+
+
+class TestSearchResults:
+    @property
+    def search_route_url(self):
+        return "/search_results_summary"
+
+    @property
+    def browse_transferring_body_route_url(self):
+        return "/search/transferring_body"
+
+    @property
+    def transferring_body_id(self):
+        return "c3e3fd83-4d52-4638-a085-1f4e4e4dfa50"
+
+    def test_search_fuzzy_search(self, standard_user_page: Page):
+        """
+        Given a search query is submitted with a typo (e.g., minor misspelling) on the
+        search_results/transferring_body/<body_id> page
+        When the results are displayed
+        Then the results should use fuzziness logic to account for the typo and display relevant matches
+        """
+        url = f"{self.browse_transferring_body_route_url}/{self.transferring_body_id}#browse-records"
+
+        standard_user_page.goto(url)
+        expect(standard_user_page).to_have_url(url)
+
+        standard_user_page.locator("#search-input").click()
+        standard_user_page.locator("#search-input").fill("smaplt")
+        standard_user_page.get_by_role("button", name="Search").click()
+        rows = standard_user_page.locator("tbody .govuk-table__row")
+        assert rows.count() == 20
+
+        found = False
+        for i in range(rows.count()):
```

Comment:
it would be good to be able to assert that it presents itself as a highlight, so within the highlight tag elements, otherwise it could just be showing that sample was in a snippet but not in the highlight